.github/workflows/test.yml

```diff
@@ -13,9 +13,6 @@ jobs:
           - 1.11-slim   # otp 23
           - 1.10-slim   # otp 22
           - 1.9-slim    # otp 22
-          - 1.8-otp-22
-          - 1.8-slim    # otp 21
-          - 1.7-slim    # otp 21
```

Comment:
Unsupported version by Elixir: https://hexdocs.pm/elixir/1.13/compatibility-and-deprecations.html